omnibus/resources/agent/msi/README.md

```diff
@@ -51,7 +51,7 @@ The following configuration command line options are available when installing t
 * DD_URL="_string_"
   * Sets the **dd_url** variable in datadog.yaml to _string_. 
 * LOGS_DD_URL="_string_"
-  * Sets the **dd_url** variable in the **logs_config** section in datadog.yaml to _string_. 
+  * Sets the **logs_dd_url** variable in the **logs_config** section in datadog.yaml to _<endpoint>:<port>_ .
```

Comment:
`_<endpoint>:<port>_` disappears after markdown formatting. You could write instead:

```suggestion
  * Sets the **logs_dd_url** variable in the **logs_config** section in datadog.yaml to _string_ . _string_ has to be of the form `<endpoint>:<format>`, for example `agent-intake.logs.datadoghq.com:443`.
```